### PR TITLE
Update the prepare_prebuilt_workers to be able to specify repo.

### PR DESCRIPTION
**Pull request by wanlin31**
The Pr provides a way for user to specify which repo they want to
build the test worker image from.

### CODE REVIEW

**Comment by wanlin31** [comment]
Close the issue since Ubuntu related ebpf tests passed recently.

File: tools/README.md [hunk 1]
```diff
@@ -91,15 +91,15 @@ the following shows this process (building and pushing prebuilt images) for
 
 ```shell
 bin/prepare_prebuilt_workers \
-     -l cxx:master \
-     -l go:master \
+     -l cxx:grpc/grpc:master \
```
Comment (paulosjca) [inline tools/README.md:94]: I don't think it is good to make a change that is not backward-compatible.

I would keep the instructions as they are now and add a section about running from a forked repo. In that section I would say that you can build from a forked repo by adding the repo in addition to the GITREF.

File: tools/cmd/prepare_prebuilt_workers/main.go [hunk 2]
```diff
@@ -37,11 +37,18 @@ import (
...
-	languagesToGitref   map[string]string
+	preBuiltImagePrefix       string
+	testTag                   string
+	dockerfileRoot            string
+	buildOnly                 bool
+	languagesToLanguageDetail map[string]LanguageDetail
+}
+
+// LanguageDetail containers the details of each tested language.
+type LanguageDetail struct {
```
Comment (paulosjca) [inline tools/cmd/prepare_prebuilt_workers/main.go:48]: LanguageDetail is a bit of an odd name. Perhaps "LanguageSpec"?

File: tools/README.md (hunk 1 above)
Comment (wanlin31) [reply tools/README.md:94]: I have updated it to make if backward-compatible, meaning it takes both cxx:master  and cxx:grpc/grpc:master, would that work? Thank you

File: tools/cmd/prepare_prebuilt_workers/main.go [hunk 3]
```diff
@@ -105,41 +112,43 @@ func main() {
 	for _, pair := range languagesSelected {
 		split := strings.Split(pair, ":")
```
[the remark this answers is not in the document]
Comment (wanlin31) [reply tools/cmd/prepare_prebuilt_workers/main.go:113]: I am sorry I missed this one.

**Pull request closed without merging**